### ISSUE TITLE
[Bug] - Tertiary button component text colour in dark mode is incorrect

### ISSUE THREAD

**Issue by ergusto** [nearform/quantum#86]

**Comment by ilteoood** [comment]
@ergusto can you provide additional details?

**Comment by ergusto** [comment]
@ilteoood We don't have a dark mode toggle in the storybook anymore as the plugin stopped working, so it's a bit difficult to test currently. As I recall, the tertiary text colour didn't match the designs and used the default white. 

I don't think there are plans to continue supporting this library however, so it may not be worth pursuing.

**Comment by simoneb** [comment]
I agree, let's not invest too much in this component library, we don't foresee it being used anywhere else at the moment.
